python/tutorials/06-fused-attention.py

```diff
@@ -137,6 +136,7 @@ def _attn_fwd_inner_tma(acc, l_i, m_i, q,  #
         # note that this non transposed v for FP8 is only supported on Blackwell
         acc = tl.dot(p, v, acc)
         # update m_i and l_i
+        l_i = l_i * alpha + l_ij
```

Comment:
Why move this? Comments seem to be duplicated

Reply:
> This also slightly tweaks the kernel code in 06-fused-attention.py to reduce register pressure. 

Because PTXAS won't do code motion across mbarrier waits and arrives, the code has to be placed carefully on the frontend. I'll leave a comment here.